tox.ini

```diff
@@ -12,7 +12,7 @@ whitelist_externals =
     sed
 deps =
     -rtest-requirements.txt
-    ansible{env:ANSIBLE_VERSION:==2.1.0}
+    ansible{env:ANSIBLE_VERSION:==2.2.2}
```

Comment:
I"m not sure why this is here to be honest. I should be able to get rid of it, right?